src/operation-node/delete-query-node.ts

```diff
@@ -22,6 +23,8 @@ export interface DeleteQueryNode extends OperationNode {
   readonly orderBy?: OrderByNode
   readonly limit?: LimitNode
   readonly explain?: ExplainNode
+  readonly frontModifiers?: ReadonlyArray<DeleteModifierNode>
+  readonly endModifiers?: ReadonlyArray<DeleteModifierNode>
```

Comment:
I don't see why these should have their own nodes. 
Should just be `ReadonlyArray<OperationNode>` for now.